noetic/distribution.yaml

```diff
@@ -5969,8 +5969,76 @@ repositories:
       test_pull_requests: true
       type: git
       url: https://github.com/ros-visualization/python_qt_binding.git
-      version: melodic-devel
+      version: noetic-devel
     status: maintained
+  qb_chain:
+    doc:
+      type: git
+      url: https://bitbucket.org/qbrobotics/qbchain-ros.git
+      version: production-noetic
+    release:
+      packages:
+      - qb_chain
+      - qb_chain_control
+      - qb_chain_controllers
+      - qb_chain_description
+      - qb_chain_msgs
+      tags:
+        release: release/noetic/{package}/{version}
+      url: https://bitbucket.org/qbrobotics/qbchain-ros-release.git
+      version: 2.2.2-1
+    source:
+      type: git
+      url: https://bitbucket.org/qbrobotics/qbchain-ros.git
+      version: production-noetic
+    status: developed
+  qb_device:
```

Comment:
This pull request relates to qb_chain. 

I've suggested that you stack your commits so that they can be merged sequentially. However I cannot see the sequence in which to merge these. They all seem to cross reference. 

Please comment and clarify how you expect them to be merged. Ideally each one will have one commit inserting one packages. If it will be easier for you. You can do each one at a time and use bloom-release with `-p` option to regenerate the additional requests after the merged PR is completed, if the git history is getting too complicated.